notebooks/src/3-create-stac-item-with-extension.ipynb

```diff
@@ -297,22 +1575,157 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Let's create a catalog, add the item that uses the EO Extension, and save to a new STAC."
+    "## View Extension\n",
+    "\n",
+    "It adds metadata related to angles of sensors and other radiance angles that affect the view of resulting data.\n",
+    "\n",
+    "![View Extension fields](../../assets/images/python-images/view_extension.png)\n",
+    "\n",
+    "Here we are adding manually the field values, but how to extract from Landsat metadata follow this [tutorial](https://github.com/stac-utils/pystac/blob/fc839fb92b628f25eb1a54cfa0317c51429c5b9b/docs/tutorials/creating-a-landsat-stac.ipynb)."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 12,
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "view_ext = ViewExtension.ext(item, add_if_missing=True)\n",
+    "view_ext.sun_azimuth = 160.86021018\n",
+    "view_ext.sun_elevation = 23.81656674"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'datetime': '2023-10-16T09:08:53.705550Z',\n",
+       " 'eo:bands': [{'name': 'Coastal',\n",
+       "   'common_name': 'coastal',\n",
+       "   'description': 'Coastal: 400 - 450 nm'},\n",
+       "  {'name': 'Blue', 'common_name': 'blue', 'description': 'Blue: 450 - 510 nm'},\n",
+       "  {'name': 'Green',\n",
+       "   'common_name': 'green',\n",
+       "   'description': 'Green: 510 - 580 nm'},\n",
+       "  {'name': 'Yellow',\n",
+       "   'common_name': 'yellow',\n",
+       "   'description': 'Yellow: 585 - 625 nm'},\n",
+       "  {'name': 'Red', 'common_name': 'red', 'description': 'Red: 630 - 690 nm'},\n",
+       "  {'name': 'Red Edge',\n",
+       "   'common_name': 'rededge',\n",
+       "   'description': 'Red Edge: 705 - 745 nm'},\n",
+       "  {'name': 'Near-IR1',\n",
+       "   'common_name': 'nir08',\n",
+       "   'description': 'Near-IR1: 770 - 895 nm'},\n",
+       "  {'name': 'Near-IR2',\n",
+       "   'common_name': 'nir09',\n",
+       "   'description': 'Near-IR2: 860 - 1040 nm'}],\n",
+       " 'platform': 'Maxar',\n",
+       " 'instruments': ['WorldView3'],\n",
+       " 'gsd': 0.3,\n",
+       " 'view:sun_azimuth': 160.86021018,\n",
+       " 'view:sun_elevation': 23.81656674}"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "item.properties"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Projection Extension\n",
+    "We are specifying the EPSG code for the image here manualy, but how to figures out the correct UTM Zone EPSG: follow this [tutorial](https://github.com/stac-utils/pystac/blob/fc839fb92b628f25eb1a54cfa0317c51429c5b9b/docs/tutorials/creating-a-landsat-stac.ipynb)\n",
```

Comment:
Reword from "We are specifying the EPSG code for the image here manualy, but how to figures out the correct UTM Zone EPSG: follow this [tutorial](https://github.com/stac-utils/pystac/blob/fc839fb92b628f25eb1a54cfa0317c51429c5b9b/docs/tutorials/creating-a-landsat-stac.ipynb)" to 
"Here, we are specifying the EPSG code for the image here manually, but PySTAC can also figure out the correct EPSG code. Check out this [tutorial](https://github.com/stac-utils/pystac/blob/fc839fb92b628f25eb1a54cfa0317c51429c5b9b/docs/tutorials/creating-a-landsat-stac.ipynb) to see how. "